ritasamir/assignment4/CountingIslands.java

```diff
@@ -0,0 +1,63 @@
+package assignment_4;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import assignment_3.Point;
+
+public class CountingIslands {
+
+	/**
+	 * @param tiles
+	 *            is a 2-dimensional array of booleans, where false means water
+	 *            and true means land
+	 * @param rowNumber
+	 *            is the number of rows
+	 * @param columnNumber
+	 *            is the number of columns
+	 * @return the number of islands.
+	 */
+	// looping through the 2d array and search for an island when we find a land

```

Comment:
nit: Move into inside of the method. Using comments with "//" is okay inside the method, but above the method there should only be the javadoc with the "/** * */" style.

Reply:
thank you, i will do it